File: 215-kth-largest-element-in-an-array/215-kth-largest-element-in-an-array.js
/**
 * @param {number[]} nums
 * @param {number} k
 * @return {number}
 */
var findKthLargest = function(nums, k) {
    
    nums.sort( (a,b) => {
        return Number(a) - Number(b);
    });
    return nums[nums.length-k];
    
};

File: 215-kth-largest-element-in-an-array/215-kth-largest-element-in-an-array.java
class Solution {
    
    public int findKthLargest(int[] nums, int k) {
        
        int n = nums.length;
        // Collections.reverseOrder()
        PriorityQueue<Integer> pq = new PriorityQueue<>(k);
        for(int i=0;i<nums.length;i++) {
            if(pq.size() == k && nums[i] <= pq.peek()) continue;
            if(pq.size() == k && nums[i] > pq.peek()) {
                pq.poll();                
            }
            pq.add(nums[i]);            
        }
        
        int answer = pq.peek();
        return answer;
    }
}